src/version/version.service.ts

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from '../prisma/prisma.service';
+import { Version } from '.prisma/client';
+
+@Injectable()
+export class VersionService {
```

Comment:
Could we use plural in the file / class name for consistency?

```suggestion
export class VersionsService {
```